lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -2022,17 +2125,22 @@ def test_certificate_regeneration_for_students(self):
                 )
 
         self.assertDictContainsSubset(
-            {
-                'action_name': 'certificates generated',
-                'total': 10,
-                'attempted': 10,
-                'succeeded': 7,
-                'failed': 3,
-                'skipped': 0,
-            },
+            expected_results,
             result
         )
 
+    def _create_students(self, students_id_start=None, students_id_end=None):
```

Comment:
once you've changed all `_create_students` calls to be zero indexed, you can just have this function be:

```python
def _create_students(self, number_of_students):
    return [
        self.create_student(
            username='student_{}'.format(index),
            email='student_{}@example.com'.format(index)
        )
        for index in xrange(number_of_students)
    ]
```